### DISCUSSION TITLE
bypassDNA... Do I understand this properly?

### DISCUSSION THREAD

**Discussion by nerdylist** [nftchef/art-engine#51] [Help + Q&A] [answered]
Hello,

Do I understand this properly... bypassDNA doesn't quite work how it does with HashLips, but instead, you just omit any rarity references in the config or filenames, so it doesn't emit rarity... These traits don't show up on metadata?

**Comment by nftchef** [comment, marked as answer]
Hi, @pstevens 
I'm not sure I understand. bypassDNA should work the same, (I wrote it into the main HashLips using the same code. Unless something has changed over there)

bypassDNA does not remove a trait from Metadata, instead, it's to be used when you don't want to include a layer as part of the "uniqueness" check. 
For example, by setting bypassDNA to the `background` layer, you would prevent any duplicates that _only have a different background_.

**Reply by nerdylist** [reply]
Thank you! :)
